cmake/modules/FindZeroMQ.cmake

```diff
@@ -62,6 +62,9 @@ find_path(ZeroMQ_INCLUDE_DIR NAMES "zmq.h" "zmq_utils.h"
     DOC "ZeroMQ include directories"
     NO_DEFAULT_PATH
 )
+if(NOT ZeroMQ_INCLUDE_DIR)
```

Comment:
You can achieve the same behaviour in a slightly more idiomatic way by dropping the `NO_DEFAULT_PATH` option of the previous [`find_path`](https://cmake.org/cmake/help/v3.0/command/find_path.html) and [`find_library`](https://cmake.org/cmake/help/v3.0/command/find_library.html) calls. See . If you could change that and add a comment at the top of the file at the paragraph near line 40, that the module will now also search the system paths - that would be awesome :)